Lib/test/test_strftime.py

```diff
@@ -39,7 +39,21 @@ def _update_variables(self, now):
         if now[3] < 12: self.ampm='(AM|am)'
         else: self.ampm='(PM|pm)'
 
-        self.jan1 = time.localtime(time.mktime((now[0], 1, 1, 0, 0, 0, 0, 1, 0)))
+        jan1 = time.struct_time(
+            (
+                now.tm_year,  # Year
+                1,  # Month (January)
+                1,  # Day (1st)
+                0,  # Hour (0)
+                0,  # Minute (0)
+                0,  # Second (0)
+                -1,  # tm_wday (will be determined)
+                1,  # tm_yday (day 1 of the year)
+                -1,  # tm_isdst (let the system determine)
+            )
+        )
+        # use mktime to get the correct tm_wday and tm_isdst values
+        self.jan1 = time.localtime(time.mktime(jan1))
```

Comment:
```suggestion
        self.jan1 = time.localtime(time.mktime((now[0], 1, 1, 0, 0, 0, -1, 1, -1)))
```
Why not just all on one line like before, it seems clear to me, and this is a test after all.

Reply:
FWIW I like the long version myself.

Reply:
IMO it is overkill and distracting, if you are looking at these tests you are probably familiar with `struct_time` anyway.